scilpy/tracking/trackable_dataset.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you use the same structure of documentation as in local_tracking

Reply:
You mean using this format?
```
Parameters
----------
[...]

Return
------
[...]
```

Reply:
Yes

Reply:
Done in ef306f3